generators/client-2/templates/src/main/webapp/app/_app.state.ts

```diff
@@ -1,5 +1,8 @@
 import { NavbarComponent } from './layouts';
-import { AuthService, <%=jhiPrefixCapitalized%>LanguageService } from './shared';
+import { AuthService } from './shared';
+<%_ if (enableTranslation) { _%>
```

Comment:
@deepu105  
Below code has condition checking for regex to be applied. What is best place to remove the import declaration.  (this is part of Generator.prototype.copyTemplate method)
` angularVersion = this.config.get('angularVersion');`
`if (angularVersion==='angular2'){`
 `// This will remove resolve translate service. If there are more services then they should be explicitly removed`
 `       regex = /\,[\s]*resolve:[\s]\[\{[\s]*[a-z]*\:[\s]*\'translate\'\,[\s]*[a-zA-z\s:\[\:\,\(\)\=\>]*\.[a-zA-z\(\)\']*[\s]*\}\]/g;`
 `      } else {`
 `         regex = /\,[\s\n ]*(resolve)\:[\s ]*[\{][\s\n ]*[a-zA-Z]+\:(\s)*\[[ \'a-zA-Z0-9\$\,\(\)\{\}\n\.\<\%\=\-\>\;\s]*\}\][\s\n ]*\}/g;`
` }`
`jhipsterUtils.copyWebResource(source, dest, regex, 'js', _this, _opt, template);`